shipstation_integration/shipstation_integration/doctype/shipstation_item_custom_field/shipstation_item_custom_field.json

```diff
@@ -0,0 +1,120 @@
+{
+ "actions": [],
+ "creation": "2023-03-22 09:59:42.179486",
+ "default_view": "List",
+ "doctype": "DocType",
+ "editable_grid": 1,
+ "engine": "InnoDB",
+ "field_order": [
+  "label",
+  "fieldtype",
+  "fieldname",
+  "length",
+  "mandatory",
+  "hidden",
+  "read_only",
+  "reqd",
+  "column_break_oh9el",
+  "options",
+  "default",
+  "fetch_from",
+  "fetch_if_empty"
+ ],
+ "fields": [
+  {
+   "bold": 1,
+   "fieldname": "label",
+   "fieldtype": "Data",
+   "in_list_view": 1,
+   "label": "Label",
+   "reqd": 1
+  },
+  {
+   "default": "0",
+   "fieldname": "mandatory",
+   "fieldtype": "Check",
+   "in_list_view": 1,
+   "label": "Mandatory"
+  },
+  {
+   "default": "0",
+   "fieldname": "hidden",
+   "fieldtype": "Check",
+   "in_list_view": 1,
+   "label": "Hidden"
+  },
+  {
+   "default": "0",
+   "fieldname": "read_only",
+   "fieldtype": "Check",
+   "in_list_view": 1,
+   "label": "Read Only"
+  },
+  {
+   "fieldname": "options",
+   "fieldtype": "Small Text",
+   "label": "Options"
+  },
+  {
+   "fieldname": "column_break_oh9el",
+   "fieldtype": "Column Break"
+  },
+  {
+   "fieldname": "default",
+   "fieldtype": "Small Text",
+   "label": "Default"
+  },
+  {
+   "fieldname": "fetch_from",
+   "fieldtype": "Small Text",
+   "label": "Fetch From"
+  },
+  {
+   "default": "0",
+   "fieldname": "fetch_if_empty",
+   "fieldtype": "Check",
+   "label": "Fetch If Empty"
+  },
+  {
+   "bold": 1,
+   "default": "Data",
+   "fieldname": "fieldtype",
+   "fieldtype": "Select",
+   "in_list_view": 1,
+   "label": "Type",
+   "options": "\nCheck\nData\nLink\nSection Break\nSelect\nSmall Text\nText",
```

Comment:
Why are we allowing Section Breaks as custom fields?

Reply:
I'm just looking to simplify the feature. I think the settings record should only be concerned about mapping a custom field from Shipstation into a field in ERPNext. Anything beyond that, like layouts, should be a framework-level concern and done separately, in my opinion.